### PR TITLE
Update Readme Game Description

### PR DESCRIPTION
**Pull request by tsudijon**
Fixed typos in description; removed latex.

### CODE REVIEW

File: README.md
Comment (DukHKim) [inline README.md:12]: Is it obvious or would it help to include note that lower score is better

File: README.md
Comment (DukHKim) [inline README.md]: For simplicity, we should set a limit for our game, e.g. We can support games of 3-8 players. Or whatever makes sense given tech?

File: README.md
Comment (DukHKim) [inline README.md]: is 1 the highest or lowest?

File: README.md
Comment (tsudijon) [reply README.md:12]: I can add "The goal of the game is to achieve the lowest group score possible"

File: README.md
Comment (tsudijon) [reply README.md]: I agree. Can discuss support for # players more in design doc.

File: README.md
Comment (tsudijon) [reply README.md]: Based on prev experience I think 1 is the highest ranked card, N would be the lowest. For example, a player with an Ace would always be first; a player with a two would always be last or tied for such

**Review by dg523 — APPROVED** [review]
LGTM, if you want you can squash these into a single commit and force push before merging

**Comment by DukHKim** [comment]
> LGTM, if you want you can squash these into a single commit and force push before merging

There's an option in the drop down next to merge pull request
also what is LGTM

**Pull request merged**
